packages/tools/locales/fr_FR.po

```diff
@@ -869,7 +869,7 @@ msgstr "date de création"
 
 #: packages/lib/Synchronizer.ts:188
 msgid "Created local items: %d."
-msgstr "Objets créés localement : %d."
+msgstr "Objets local créés : %d."
```

Comment:
"objet locaux" ?

Reply:
sounds a bit strange for me but why not. It still shorter and will be displayed in oneline. In fact with "local" I was thinking to the environnement (local machine) but it may not be ideal indeed.

Reply:
Hmm, yes let's go for "locaux" or else maybe something like "Object créés (local)" but it means we'd need to change all the other instances to match this style.